--- a/src/Makefile	2024-09-26 14:42:27.124575709 -0400
+++ b/src/Makefile-new	2024-09-26 19:30:10.145699461 -0400
@@ -2410,10 +2410,10 @@
 	libcmd-serv.la libcmd-cli.la libcmd-cli-debug.la \
 	libcmd-certtool.la libcmd-danetool.la $(am__append_15) \
 	$(am__append_16) libcmd-systemkey.la libcerttool-cfg.la
-#srptool_SOURCES = srptool.c
-#srptool_LDADD = ../lib/libgnutls.la libcmd-srp.la \
-#	../gl/libgnu.la $(LTLIBINTL) gl/libgnu_gpl.la
-#libcmd_srp_la_SOURCES = srptool-options.c srptool-options.h
+srptool_SOURCES = srptool.c
+srptool_LDADD = ../lib/libgnutls.la libcmd-srp.la \
+	../gl/libgnu.la $(LTLIBINTL) gl/libgnu_gpl.la
+libcmd_srp_la_SOURCES = srptool-options.c srptool-options.h
 psktool_SOURCES = psk.c
 psktool_LDADD = ../lib/libgnutls.la libcmd-psk.la ../gl/libgnu.la \
 	$(LTLIBINTL) gl/libgnu_gpl.la
@@ -2481,15 +2481,15 @@
 	../lib/libgnutls.la gl/libgnu_gpl.la \
 	../gl/libgnu.la $(LTLIBREADLINE) \
 	$(INET_PTON_LIB) $(LIB_CLOCK_GETTIME)
-#tpmtool_SOURCES = tpmtool-options.json tpmtool.c certtool-common.c certtool-extras.c common.c
-#tpmtool_LDADD = ../lib/libgnutls.la \
-#	libcmd-tpmtool.la ../gl/libgnu.la \
-#	gl/libgnu_gpl.la $(COMMON_LIBS)
-#libcmd_tpmtool_la_SOURCES = tpmtool-options.c tpmtool-options.h
-#libcmd_tpmtool_la_LIBADD = libcerttool-cfg.la \
-#	../lib/libgnutls.la gl/libgnu_gpl.la \
-#	../gl/libgnu.la $(LTLIBREADLINE) \
-#	$(INET_PTON_LIB) $(LIB_CLOCK_GETTIME)
+tpmtool_SOURCES = tpmtool-options.json tpmtool.c certtool-common.c certtool-extras.c common.c
+tpmtool_LDADD = ../lib/libgnutls.la \
+	libcmd-tpmtool.la ../gl/libgnu.la \
+	gl/libgnu_gpl.la $(COMMON_LIBS)
+libcmd_tpmtool_la_SOURCES = tpmtool-options.c tpmtool-options.h
+libcmd_tpmtool_la_LIBADD = libcerttool-cfg.la \
+	../lib/libgnutls.la gl/libgnu_gpl.la \
+	../gl/libgnu.la $(LTLIBREADLINE) \
+	$(INET_PTON_LIB) $(LIB_CLOCK_GETTIME)
 systemkey_SOURCES = systemkey.c certtool-common.c common.c
 systemkey_LDADD = ../lib/libgnutls.la libcmd-systemkey.la \
 	../gl/libgnu.la gl/libgnu_gpl.la $(COMMON_LIBS)
